appveyor.yml

```diff
@@ -16,6 +16,8 @@ skip_tags: true
 environment:
   MARKTEXT_IS_STABLE: 1
   MARKTEXT_EXIT_ON_ERROR: 1
+  GITHUB_TOKEN:
```

Comment:
Why add `GITHUB_TOKEN`? What does it use for?

Reply:
Same as Travis CI: vscode-ripgrep API limit on GitHub